### PR TITLE
Update release-published.yml

### PR DESCRIPTION
**Pull request by sayaliM0412**
fix: `.github/workflows/release-published.yml`: do not deploy to MavenCentral
NOTE: if you don't push to sonatype, you don't release to maven

### CODE REVIEW

File: .github/workflows/release-published.yml [hunk 1]
```diff
@@ -8,4 +8,6 @@ on:
 jobs:
   release:
     uses: liquibase/build-logic/.github/workflows/extension-release-published.yml@v0.7.8
```
Comment (jnewton03) [inline .github/workflows/release-published.yml:10]: @jandroav @sayaliM0412 if we are merging the new input into the `main` branch, do we need to change line 10 to be @main ?

Comment (sayaliM0412) [reply .github/workflows/release-published.yml:10]: I was thinking of re-releasing `v0.7.8` with these changes.

Comment (sayaliM0412) [reply .github/workflows/release-published.yml]: I just changed it to using @main

**Comment by sayaliM0412** [comment]
we need the `PRO_LICENSE_KEY`, `GHA_AWS_KEY_ID`, `GHA_AWS_KEY`  for Build and Test to work as the workflow uses pro-extension-test.yml from build-logic : https://github.com/liquibase/build-logic/blob/main/.github/workflows/pro-extension-test.yml. Currently, all of them are available to "Private and internal repositories". 

cc: @jnewton03 @StevenMassaro

**Comment by jnewton03** [comment]
> we need the `PRO_LICENSE_KEY`, `GHA_AWS_KEY_ID`, `GHA_AWS_KEY` for Build and Test to work as the workflow uses pro-extension-test.yml from build-logic : https://github.com/liquibase/build-logic/blob/main/.github/workflows/pro-extension-test.yml. Currently, all of them are available to "Private and internal repositories". For public repositories, we usually have an authorize step to allow for a run from external contributors and then manually add "Selected repositories" for private repositories to work. Does that sound good?
> 
> cc: @jnewton03 @StevenMassaro

We should follow the same pattern we use with other public repo extensions such as neo4j

**Comment by sayaliM0412** [comment]
> > we need the `PRO_LICENSE_KEY`, `GHA_AWS_KEY_ID`, `GHA_AWS_KEY` for Build and Test to work as the workflow uses pro-extension-test.yml from build-logic : https://github.com/liquibase/build-logic/blob/main/.github/workflows/pro-extension-test.yml. Currently, all of them are available to "Private and internal repositories". For public repositories, we usually have an authorize step to allow for a run from external contributors and then manually add "Selected repositories" for private repositories to work. Does that sound good?
> > cc: @jnewton03 @StevenMassaro
> 
> We should follow the same pattern we use with other public repo extensions such as neo4j

we do the same thing for all the public repositories. In this case as we are using `pro-extension-test.yml`, we would need to add the keys mentioned above to "Selected repositories" instead of "All" or "Private and internal repositories"

**Comment by jnewton03** [comment]
@sayaliM0412 yes, I see.  This is the first `pro` extension that is public.

**Comment by sayaliM0412** [comment]
> GHA_AWS_KEY

added. Job running: https://github.com/liquibase/liquibase-aws-license-service/actions/runs/10168951873?pr=13

**Comment by sayaliM0412** [comment]
we dont actually need `.github/workflows/release-published.yml`. We are already running https://github.com/liquibase/liquibase-aws-license-service/blob/main/.github/workflows/create-release.yml and https://github.com/liquibase/liquibase-aws-license-service/blob/main/.github/workflows/attach-artifact-release.yml

**Pull request merged**